src/collective/cover/behaviors/interfaces.py

```diff
@@ -25,10 +26,16 @@ class IRefresh(model.Schema):
         default=300,
     )
 
-alsoProvides(IRefresh, form.IFormFieldProvider)
+alsoProvides(IRefresh, IFormFieldProvider)
 
 
-@form.validator(field=IRefresh['ttl'])
-def validate_ttl(value):
-    if value <= 0:
-        raise Invalid(_(u'Value must be greater than zero.'))
+class TimeToLiveValidator(validator.SimpleFieldValidator):
+
+    """z3c.form validator class for time to live"""
+
+    def validate(self, value):
```

Comment:
@hvelarde it seems to me correct. See: http://docs.plone.org/develop/addons/schema-driven-forms/customising-form-behaviour/validation.html#field-widget-validators